Assets/Mirror/Editor/Weaver/Processors/RpcProcessor.cs

```diff
@@ -68,6 +68,12 @@ public static MethodDefinition ProcessRpcCall(TypeDefinition td, MethodDefinitio
 
             NetworkBehaviourProcessor.WriteSetupLocals(worker);
 
+            if (Weaver.GenerateLogErrors)
+            {
+                worker.Append(worker.Create(OpCodes.Ldstr, "Call ClientRpc function " + md.Name));
+                worker.Append(worker.Create(OpCodes.Call, Weaver.logErrorReference));
+            }
+
```

Comment:
what is this error for?

Reply:
Copied from CommandProcessor
```cs
public static MethodDefinition ProcessCommandCall(TypeDefinition td, MethodDefinition md, CustomAttribute commandAttr)
{
    MethodDefinition cmd = MethodProcessor.SubstituteMethod(td, md, "Call" + md.Name);

    ILProcessor worker = md.Body.GetILProcessor();

    NetworkBehaviourProcessor.WriteSetupLocals(worker);

    if (Weaver.GenerateLogErrors)
    {
        worker.Append(worker.Create(OpCodes.Ldstr, "Call Command function " + md.Name));
        worker.Append(worker.Create(OpCodes.Call, Weaver.logErrorReference));
    }
```

Reply:
It seems a bit like an error to check if a method is being called while debugging. 

Nothing in our code sets `Weaver.GenerateLogErrors`

maybe @vis2k knows more?